### ISSUE TITLE
question about the segformer baseline

### ISSUE THREAD

**Issue by ydhongHIT** [GuoleiSun/VSS-CFFM#10]
Thanks for the great work. I notice that you provide the configs of segformer on VSPW. Is this the baseline you use in your paper? But I can't find the implementation of 'SegFormerHead_clips_resize1_8_focal_trans3'. In addition, the dataset configs are different for this baseline and your CFFM.

**Comment by GuoleiSun** [comment]
Hello, thanks for you interest. The configs of segformer on vspw are not for baselines. For baselines, you can simply run the configs of CFFM, but you need to change the loss in [this place](https://github.com/GuoleiSun/VSS-CFFM/blob/8537cb54769fbdd5e67c75d4835bb3d90d0318e9/mmseg/models/decode_heads/decode_head.py#L825). From line 825 to line 833, change them to
```
loss['loss_seg'] = self.loss_decode(
            seg_logit_ori,
            seg_label_ori,
            weight=seg_weight,
            ignore_index=self.ignore_index)
```
This means to comment out the loss for our video module, while only keeping the the original losses (for each separate frame). In this way, all the training settings for the baselines are kept the same as CFFM. Thus the comparisons are fair. Please let me know you have more questions.
